source/upgrade/manager.rst

```diff
@@ -2,6 +2,9 @@
 Manager
 =======
 
+Preparations
+============
```

Comment:
--------- instead of ========= (it a sub section of the manager section)

Reply:
Fixed.